modules/gapi/src/streaming/onevpl/cfg_params_parser.cpp

```diff
@@ -116,6 +127,32 @@ mfxVariant cfg_param_to_mfx_variant(const CfgParam& cfg_val) {
             }), cfg_val.get_value());
     return ret;
 }
+
+size_t strtoull_or_throw(const char* str) {
```

Comment:
y use chars

Reply:
to satisty c-string in  interface functions (to make interface ABI consistent)